### DISCUSSION TITLE
Any Jupyter Notebooks or CoLaboratory

### DISCUSSION THREAD

**Discussion by defencedog** [chmarti1/PYroMat#51] [Ideas]
It will be highly helpful & accelerate learning curve if author can provide some `jupyter notebooks`. [PDF handbook ](http://pyromat.org/pdf/handbook.pdf)is great but some solved examples can really add learning substance

**Comment by chmarti1** [comment]
I read this comment right away, but I've been slow to reply - I'm sorry about that.  I agree that solved examples are really helpful.  It might be useful to have a couple of common analysis problems worked out using Python code.  I'll keep that on my to-do list for the next edition.

You might also be excited to know that we are already working on improved user interface ideas.  Instead of Jupyter, we're trying to make PYroMat more accessible to people who do not use Python, but it will take some time before we will be able to announce anything.

Thanks so much for your interest, and thanks so much for your comments!
